src/main/webapp/app/exercises/modeling/assess/modeling-assessment-editor/modeling-assessment-editor.component.ts

```diff
@@ -200,13 +203,14 @@ export class ModelingAssessmentEditorComponent implements OnInit {
             return;
         }
 
-        const generalFeedbackIndex = feedback.findIndex((feedbackElement) => feedbackElement.reference == null);
+        const generalFeedbackIndex = feedback.findIndex((feedbackElement) => feedbackElement.reference == null && feedbackElement.type !== FeedbackType.MANUAL_UNREFERENCED);
         if (generalFeedbackIndex >= 0) {
             this.generalFeedback = feedback[generalFeedbackIndex] || new Feedback();
             feedback.splice(generalFeedbackIndex, 1);
         }
 
-        this.referencedFeedback = feedback;
+        this.referencedFeedback = feedback.filter((feedbackElement) => feedbackElement.reference != null);
+        this.unreferencedFeedback = feedback.filter((feedbackElement) => feedbackElement.reference == null && feedbackElement.type === FeedbackType.MANUAL_UNREFERENCED);
```

Comment:
`feedbackElement.reference == null` should be redundant here, since `FeedbackType.MANUAL_UNREFERENCED` already implies that, right?